Vector2D position
Vector2D velocity
Vector2D acceleration
